SabaCenter/View/CustomControls/ScheduleItemCell.swift

```diff
@@ -24,10 +24,17 @@ class ScheduleItemCell: UITableViewCell {
         // Configure the view for the selected state
     }
 
-    func updateCell(schedule: ScheduleViewModel) {
+    func update(schedule: ScheduleViewModel) {
```

Comment:
Why not just path the `indexPath` in and use that to determine the color, rather than tagging it?